implementations/rust/ockam/ockam_api/schema.cddl

```diff
@@ -190,3 +190,42 @@ request_enrollment_token = {
     ?0: 8560526,
      1: attributes
 }
+
+;;; Signer ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+sign_request = attributes
+
+credential = {
+    ?0: 3796735,
+     1: bytes, ;; CBOR-encoded attributes
+     2: signer_signature
+}
+
+signer_signature = [
+    0: identity_id,
+    1: signer_signature_bytes
+]
+
+signer_signature_bytes = bytes
+
+add-identity = {
+    ?0: 1544679,
+     1: identity
+}
+
+get-identity-response = {
+    ?0: 9967409,
+     1: identity
+}
+
+;;; Authenticator ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
You mean splitting `schema.cddl`?

Reply:
Yes

Reply:
Yes, it would be possible. The functions `assert_request_match` and `assert_response_match` would need to accept the schema file as another parameter.